resources/resources.go

```diff
@@ -33,7 +36,11 @@ func InstallPostgres(resourceDir, pgDataDir, pgUser string) (string, error) {
 	case "darwin":
 		version, data = "postgres-darwin-amd64-14.2.0", postgresDarwin
 	case "linux":
-		version, data = "postgres-linux-amd64-14.2.0", postgresLinux
+		if isAlpineLinux() {
```

Comment:
If there is no easy way to tell which distro we use, then we can add a log for the case block

Reply:
Done. Logged file name instead.